app/src/utils/variableProviders.ts

```diff
@@ -0,0 +1,193 @@
+import { Breakdowns, Dataset, Row } from "./DatasetTypes";
+
+const USA_STRING = "the USA";
```

Comment:
I also have this string in Fips.ts STATE_FIPS_MAP[0]

not sure if it's worth aligning those (in case we're matching on them)

Reply:
Updated to STATE_FIPS_MAP[0], I think it's worth standardizing instead of using magic strings. I think we'll want this to come with datasets eventually.